src/formulas/tokenizer.ts

```diff
@@ -196,10 +196,19 @@ function tokenizeSymbol(chars: string[]): Token | null {
   return null;
 }
 
-const whiteSpaceRegexp = /\s/;
+const whiteSpaceRegexp = / |\t|\v|\f/;
+const newLineRegex = /\n|\r/;
```

Comment:
No need to handle `\r` or `\r\n` newline characters, we remove them in `cell.ts`. 
Should we do that for the other spaces characters ? What the hell are we gonna do with Form Feeds, or Vertical Tabs ?

Reply:
Changed a bit how we handle that : at the start of `tokenize()` I replace all the different spaces/newlines by simple spaces and "\n", so the handling in the composer is very simple.